Schedule.API/DTOs/RoomRenovationDto.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Schedule.API.Model.Utilities;
+
+namespace Schedule.API.DTOs
+{
+    public class RoomRenovationDto
+    {
+        public int FirstRoomId { get; set; }
+        public int SecondRoomId { get; set; }
```

Comment:
Better name them Source and Destination Room

Reply:
Fixed